scripts/devbuild_ofp_odp_dpdk.sh

```diff
@@ -27,21 +27,20 @@ sed -ri 's,(CONFIG_RTE_LIBRTE_PMD_PCAP=).*,\1y,' .config
 popd
 
 #Build DPDK
-make -j${JOBS} build O=${TARGET} EXTRA_CFLAGS="-fPIC"
-make install O=${TARGET} DESTDIR=${TARGET}
+make -j${JOBS} install T=${TARGET} DESTDIR=./install EXTRA_CFLAGS="-fPIC"
 popd
 
 # Clone odp-dpdk
 git clone -q https://github.com/Linaro/odp-dpdk
```

Comment:
This can be updated to: https://github.com/OpenDataPlane/odp-dpdk

Reply:
OK